plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -76,19 +76,23 @@ func init() {
 	// PodFitsPorts has been replaced by PodFitsHostPorts for better user understanding.
 	// For backwards compatibility with 1.0, PodFitsPorts is regitered as well.
 	factory.RegisterFitPredicate("PodFitsPorts", predicates.PodFitsHostPorts)
+	// Fit is defined based on the absence of port conflicts.
```

Comment:
Add to the comment: "This predicate is actually a default predicate, because it is invoked from predicates.GeneralPredicates()"

(otherwise people might assume all of the predicate functions mentioned in init() are not default predicates)